webapp/pages/profile/_id/_slug.vue

```diff
@@ -85,6 +85,10 @@
             </ds-space>
           </template>
         </ds-card>
+        <!-- User ID -->
+        <ds-text color="softer" size="small" class="hyphenate-text" style="text-align: center;">
+          <small>{{ $t('profile.userId') }}: {{ user.id }}</small>
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T18:45:27Z" title="Thursday, January 23rd 2020, 7:45:27 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -85,6 +85,10 @@
             </ds-space>
           </template>
         </ds-card>
+        <!-- User ID -->
+        <ds-text color="softer" size="small" class="hyphenate-text" style="text-align: center;">
+          <small>{{ $t('profile.userId') }}: {{ user.id }}</small>
```


Can we move it into the user card? I think it belongs to the user details.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-24T08:55:40Z" title="Friday, January 24th 2020, 9:55:40 am +01:00">Jan 24, 2020</time>_

---


I understand your thinking @roschaefer !
But for me it is ugly additional technical information and therefore I kind of hide it.

What does the other @Human-Connection/hc-team and especially @alina-beck say?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-27T11:01:40Z" title="Monday, January 27th 2020, 12:01:40 pm +01:00">Jan 27, 2020</time>_

---


as I posted in zoom chat
```
I guess since I didn't see any planning on this issue, I would question if there isn't another way of dealing with the user issue

Like would having it somewhere display on hover be sufficient

is there a way that we can shorten the ids, and still have them be unique, as you have pointed out? then it might look better than these super long ids that can easily break our design...
```

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-27T11:59:19Z" title="Monday, January 27th 2020, 12:59:19 pm +01:00">Jan 27, 2020</time>_

---


I'm with @mattwr18, I think we should have a closer look at _what exactly_ we are trying to solve with displaying the `user IDs`, _where_ we need them and _how_ users should be able to interact with them (e.g. toggle them on `click` or `hover`, in the `settings`, have them visible all the time...) and _then_ make some design decisions.

There's also this other issue dealing with `user IDs` that is closely related: #2784. And then I remember that @Mogge pointed out that the `IDs` we're using right now are not really humanly readable and thus not useful for anyone, so maybe #2786 should take priority over other `user ID` issues (also because it might make the design easier).

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-28T21:33:26Z" title="Tuesday, January 28th 2020, 10:33:26 pm +01:00">Jan 28, 2020</time>_

---


The problem is that people make screenshots and send those as email as some sort of evidence for guideline violations to our moderation team. The IDs are in the URL and you cannot see them on the screenshots. So, `hover` would not make a lot of sense here.

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-29T15:35:13Z" title="Wednesday, January 29th 2020, 4:35:13 pm +01:00">Jan 29, 2020</time>_

---


So we're defining a feature for users who refuse to use our built-in reporting system? :grin:

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-03T09:04:50Z" title="Monday, February 3rd 2020, 10:04:50 am +01:00">Feb 3, 2020</time>_

---


It’s more than that:
On PlanetRomeo I had several times the experience in my live that I felt this user was already somewhen contacting me under a different name. So I made a screenshot which included their IDs at that time. Later I was able to prove for me and others that I was right.

The hover does not work for several screen shot tools I was told by @DennisHack and others …

To shorten the ID(s) there are already old issues I made long ago:

- Slugs - Permanent, unbreakable, readable and text changeable slugs for users, posts etc. #209
- 🚀 [Feature] Show User ID On User Profile Page #252

Which I have extracted into:

- 🚀 [Feature] Nano IDs For Human Readability
#2786

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-03T09:06:09Z" title="Monday, February 3rd 2020, 10:06:09 am +01:00">Feb 3, 2020</time>_

---


I would love to have a general discussion about it @Human-Connection/hc-dev-team !